process_preds.Rmd

```diff
@@ -74,11 +87,18 @@ process_preds <- function(predictor_path, template, resolution, distance=FALSE,
 
     # Set template temporarily to raster CRS
     shapefile_temp <- st_transform(template, crs = crs(raster_file))
-    raster_file <- crop(raster_file, shapefile_temp)
     
+    # crop to the temp shapefile
+    raster_file <- crop(raster_file, shapefile_temp)
+  
     # Project cropped raster into template CRS
     raster_file <- project(raster_file, crs(template), res = resolution)
```

Comment:
Instead of adding the new code chunk below, there might be some arguments you could add to `project()` to fix the issue. If you haven't already, look into these arguments and see if we can make this a little more streamlined.